get_data/utils/s3_utils.py

```diff
@@ -40,16 +57,18 @@ def upload_to_s3_from_label(l_label, s3_bucket_name, prefix="", overwrite=False)
     s3 = get_s3_resource()
     already_exist_pic = []
     upload_success = []
-    for label in l_label:
+    log = ""
+    for label in tqdm(l_label):
         picture = Path(label["location"]) / label["file_name"]
         pic_id = label["img_id"]
         key = prefix + pic_id
         if not overwrite and file_exist_in_bucket(s3, s3_bucket_name, key):
-            print(f'  --> Can\'t upload file "{picture}" because key "{key}" already exists in bucket "{s3_bucket_name}"')
+            log += f'  --> Can\'t upload file "{picture}" because key "{key}" already exists in bucket "{s3_bucket_name}"\n'
```

Comment:
je comprends pas pk tu mets les {} entre guillemets. f ne le transforme pas directement ?

Reply:
c'est juste pour print les noms entre guillemet. Par ex: 
--> Can't upload file "20201119T20-10-03-014569.jpg" because key "20201119T20-10-03-014569" already exits...
Question de gout, je trouve ça plus propre comme output (notamment quand il y a des espaces au debut/fin des variables qui ne se voit pas bien quand tu print)